.github/dependabot.yml

```diff
@@ -2,6 +2,16 @@
 
 version: 2
 updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    ignore:
+       # Only allow for patch release upgrades
+      - dependency-name: '*'
+        update-types: ['version-update:semver-major', 'version-update:semver-minor']
```

Comment:
What's up with mixing single quotes and double quotes in the same yaml file? 😆

Reply:
That is a copy/paste issue :) We can fix it, but for whatever reason I do it all the time.